core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy

```diff
@@ -0,0 +1,44 @@
+package jenkins.tools.GlobalToolConfiguration
+
+import hudson.Functions
+import hudson.model.Descriptor
+
+def f=namespace(lib.FormTagLib)
+def l=namespace(lib.LayoutTagLib)
+def st=namespace("jelly:stapler")
+
+l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName) {
+    l.side_panel {
+        l.tasks {
+            l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
+            l.task(icon:"icon-setting icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"))
+        }
+    }
+    l.main_panel {
+        h1 {
+            l.icon(class: 'icon-setting icon-xlg')
+            // TODO more appropriate icon
```

Comment:
Well, I'm not a good icon designer. If someone sends one, I'll use it.
Anyway, now that _Manage Jenkins_ is using `icon-gear` I think it's ok to keep `icon-setting` here.